ml/classification_utils.py

```diff
@@ -17,9 +17,14 @@
 
 
 def fetch_image(image_id):
-    """Gets the image from the specified ID. It returns only images
-    downloaded in the folder specified in the configuration object."""
+    """Gets the image from the specified ID. As default behaviour it
+    returns only images downloaded in the folder specified in the
+    configuration object.
+    If the default path doesn't exist the image might have been uploaded by the user."""
     image_path = os.path.join(conf.image_folder_path, image_id)
+
+    if not os.path.exists(image_path):
```

Comment:
what if the user uploads an image with the same name as one of the images in the image folder? I know this is unlikely, but we cannot exclude it completely.

Reply:
done